config/application.rb

```diff
@@ -6,6 +6,7 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "sprockets/railtie"
+require "sidekiq/api"
```

Comment:
I don't know much about sidekiq. Is this line required? Do we make use of any of these query methods in the code?